### PR TITLE
Functional Testing Unblocking

### PR DESCRIPTION
**Pull request by wlathrop21**
## Summary

*Skip "Huge Test Set" tests as it is too large to be run in the pipeline.

*Added a wait to fix potential issue in UI testing.

### CODE REVIEW

File: test/functional/apps/management/_test_huge_fields.js [hunk 1]
```diff
@@ -4,13 +4,14 @@ export default function ({ getService, getPageObjects }) {
   const esArchiver = getService('esArchiver');
   const PageObjects = getPageObjects(['common', 'home', 'settings']);
 
-  describe('test large number of fields @skipcloud', function () {
+  describe.skip('test large number of fields @skipcloud', function () {
```
Comment (lulei2007) [inline test/functional/apps/management/_test_huge_fields.js:9]: pls add comment as well:
// Continuously Failed, data is too large
// Backlog: https://sim.amazon.com/issues/P50402631

File: test/functional/apps/management/_test_huge_fields.js [hunk 2]
```diff
@@ -4,13 +4,14 @@ export default function ({ getService, getPageObjects }) {
...
     const EXPECTED_FIELD_COUNT = '10006';
     before(async function () {
       await esArchiver.loadIfNeeded('large_fields');
       await PageObjects.settings.navigateTo();
       await PageObjects.settings.clickKibanaIndices();
       await PageObjects.settings.createIndexPattern('testhuge', 'date');
+
```
Comment (lulei2007) [inline test/functional/apps/management/_test_huge_fields.js]: remove un-necessary change

File: test/functional/apps/dashboard/_dashboard.js [hunk 3]
```diff
@@ -44,6 +46,7 @@ export default function ({ getService, getPageObjects }) {
     });
 
     it('set the timepicker time to that which contains our test data', async function setTimepicker() {
+      await sleep(60000);
```
Comment (lulei2007) [inline test/functional/apps/dashboard/_dashboard.js]: why wait 60 seconds

File: test/functional/apps/management/_test_huge_fields.js (hunk 1 above)
Comment (wlathrop21) [reply test/functional/apps/management/_test_huge_fields.js:9]: Completed

File: test/functional/apps/management/_test_huge_fields.js (hunk 2 above)
Comment (wlathrop21) [reply test/functional/apps/management/_test_huge_fields.js]: Removed

**Review by lulei2007 — APPROVED** [review]

**Pull request merged**